gusto/time_discretisation/wrappers.py

```diff
@@ -6,9 +6,10 @@
 
 from abc import ABCMeta, abstractmethod
 from firedrake import (
-    FunctionSpace, Function, BrokenElement, Projector, Interpolator,
-    Constant, as_ufl, dot, grad, TestFunction, MixedFunctionSpace
+    FunctionSpace, Function, BrokenElement, Projector, Constant, as_ufl, dot, grad,
+    TestFunction, MixedFunctionSpace
```

Comment:
```suggestion
    TestFunction, MixedFunctionSpace, assemble
```
Ah sorry, we also need to import `assemble` here now!